lessons/requirements.md

```diff
@@ -0,0 +1,14 @@
+## Requirements
+
+This codelab is for anyone who are getting started with developing Firefly apps. There are a few prerequisites to get you started.
+
+* Project Firefly
+  * Please make sure that your local machine is equipped with [Technical Requirements](https://github.com/AdobeDocs/project-firefly/blob/master/getting_started/technical_requirements.md).
+  * It is useful that you are familiar with Project Firefly tooling. Therefore you are recommended to go though ["Getting Started with Project Firefly"](https://github.com/AdobeDocs/project-firefly/blob/master/getting_started/setup.md) before starting this lab.
+
+* Campaign Standard APIs
+  * You should have sufficient access to an Adobe Campaign Standard instance as well as an organization on [Adobe I/O Console](https://console.adobe.io/) provisioned with Campaign Standard API in order to work on this lab.
+  * We assume you have an available integration on I/O Console with enabled Campaign Standard API service
+  * TODO: self-provisioning might be provided.
```

Comment:
what do you have in mind for self provisioning ?

Reply:
It is about temp access to an org or integration so that someone can connect to an ACS instance without having one. Is it still on the table or totally up to the learners?

Reply:
on the table ... so we have to find an alternative solution for now.

Reply:
afaiu, you have to have access to an org with eg provisioned ACS api.

Isn't line #11 a duplicate of #10? Or do you mean "created a project already"?

Reply:
@icaraps : ok then let's assume the readers already have access to an org... I will delete the to-do.
@schaulinsan : yeah I mean "created a project already", as we don't want to guide people to create a console integration in this lab (should have been covered by a screencast already, which can be linked to once it's published)